fury/ui/core.py

```diff
@@ -1078,6 +1046,71 @@ def background_color(self, color):
             self.background.set_visibility(True)
             self.background.color = color
 
+    def update_alignment(self):
+        """Update Text Alignment.
+        """
+        text_property = self.actor.GetTextProperty()
+        updated_text_position = [0, 0]
+
+        if self.justification == 'left':
+            text_property.SetJustificationToLeft()
+            updated_text_position[0] = self.boundingbox[0]
+        elif self.justification == 'center':
+            text_property.SetJustificationToCentered()
+            updated_text_position[0] = self.boundingbox[0] + \
+                (self.boundingbox[2]-self.boundingbox[0])//2
+        elif self.justification == 'right':
+            text_property.SetJustificationToRight()
+            updated_text_position[0] = self.boundingbox[2]
+        else:
+            msg = 'Text can only be justified left, right and center.'
+            raise ValueError(msg)
+
+        if self.vertical_justification == 'bottom':
+            text_property.SetVerticalJustificationToBottom()
+            updated_text_position[1] = self.boundingbox[1]
+        elif self.vertical_justification == 'middle':
+            text_property.SetVerticalJustificationToCentered()
+            updated_text_position[1] = self.boundingbox[1] + \
+                (self.boundingbox[3]-self.boundingbox[1])//2
+        elif self.vertical_justification == 'top':
+            text_property.SetVerticalJustificationToTop()
+            updated_text_position[1] = self.boundingbox[3]
+        else:
+            msg = 'Vertical justification must be: bottom, middle or top.'
+            raise ValueError(msg)
+
+        self.actor.SetPosition(updated_text_position)
+
+    def cal_size_from_message(self):
+        "Calculate size of background according to the message it contains."
+        lines = self.message.split("\n")
+        max_length = max(len(line) for line in lines)
+        return [max_length*self.font_size, len(lines)*self.font_size]
+
+    def update_bounding_box(self, size=None):
+        """Update Text Bounding Box.
+
+        Parameters
+        ----------
+        size : (int, int) or None
+            If None, calculates bounding box.
+            Otherwise, uses the given size.
+
+        """
+        if size is None:
+            size = self.cal_size_from_message()
+
+        self.boundingbox = [self.position[0], self.position[1],
```

Comment:
I'm getting this comment in my editor, so I wonder if this is a good way to handle `boundingbox` variable, as it is not an attribute from the class itself but you are using _self._ to define it.

![image](https://github.com/fury-gl/fury/assets/31288525/1dc27b16-f8e1-4ecd-80ff-1a64d0df30ec)